vault/changelog.release.2022-02-15.md

```diff
@@ -0,0 +1,77 @@
+---
+id: egxM8X43CqrQLuhYQ6Wu8
+title: '0.82'
+desc: ''
+updated: 1644904313283
+created: 1644902103166
+---
+
+Dendron 0.82 has sprouted  🌱
+
+Schema template variables can now include `CURRENT_WEEK`.
+
+Example:
+
+```md
+Today is <%=CURRENT_YEAR%>.<%= CURRENT_MONTH%>.<%= CURRENT_DAY%>
+Week <%=CURRENT_WEEK%>
+
+Weekly goals: [[weekly.journal.<%=CURRENT_YEAR%>.week.<%=CURRENT_WEEK%>]]
+```
+
+When the template is applied by schema, it renders to:
+
+```md
+Today is 2022.02.15
+Week 07
+
+Weekly goals: [[weekly.journal.2022.week.07]]
+```
+
+### Highlights
+- enhance(schemas): Extend date subsitution to include week numbers ([[docs|dendron://dendron.dendron-site/dendron.topic.templates.schema-template#template-variables]])
+
+### Everything Else
+- fix(workspace): avoid workspace watcher crashing if folder is deleted
+- fix(workspace): correct title generation of notes within sub-hierarchy starting with `md`
+- fix(pod): acknowledge cli args for publish pod
+
+### Community
+
+#### General Updates
+
+- What strategies do you use to maintain and take care of existing content in your digital gardens? We've added a new workflow all about how we prune and repot notes throughout our documentation: [[Pruning and Repotting|dendron://dendron.dendron-site/dendron.guides.workflows.pruning-and-repotting]]
+- Thanks to all the people that have taken the 2022 dendron user survey. If you would like to help us improve dendron, please checkout the survey (and earn a shiny discord badge in the process)!
+  - [2022 Dendron User Survey](https://link.dendron.so/74EI)
```

Comment:
can we add this into its own section under "Thank You's"

**Dendron User Survey**
A huge thanks to the following Dendrologist that provided feedback in the [2022 Dendron User Survey](https://link.dendron.so/74EI)). If you would like to help us improve dendron, please checkout the survey (and earn a shiny discord badge in the process)!